lib/everest.cpp

```diff
@@ -188,7 +188,8 @@ json Everest::call_cmd(const Requirement& req, const std::string& cmd_name, json
     }
 
     if (this->validate_data_with_schema) {
-        json_validator validator(Config::loader, Config::format_checker);
+        json_validator validator([this](const json_uri& uri, json& schema) { this->config.ref_loader(uri, schema); },
+                                 Config::format_checker);
```

Comment:
I think it's also possible to extend the json_validator and use that as the validator object here.  Might be cleaner and more flexible then the lambda here